treeherder/log_parser/parsers.py

```diff
@@ -319,6 +319,11 @@ def parse_line(self, line, lineno):
                 title, value = line.split("<br/>", 1)
                 artifact["title"] = title
                 artifact["value"] = value
+            # and split TinderboxPrint or similar long lines
+            elif "href" in line and "title" in line:
```

Comment:
Cool, yah if anyone changes the output content/order in printbuildrev, that would likely happen. No probs, thanks @camd!